src/main/scala/com/webtrends/harness/component/akkahttp/AkkaHttpBase.scala

```diff
@@ -167,7 +151,7 @@ trait AkkaHttpBase extends  PathDirectives with MethodDirectives with AccessLog
                             case Success(akkaResp: AkkaHttpCommandResponse[T]) =>
                               val codeResp = akkaResp.copy(statusCode = Some(defaultCodes(outputBean, akkaResp)))
                               val finalResp = beforeReturn(outputBean, codeResp)
-                              logAccess(request, outputBean, finalResp.statusCode)
+                              //logAccess(request, outputBean, finalResp.statusCode)
```

Comment:
Was there a reason why we needed to disable this logging? I know that DAPI uses this feature to check who has accessed our APIs

Reply:
no more custom command objects so this whole file is gone. Marking access logging as an open issue.